src/foam/nanos/auth/Address.js

```diff
@@ -219,27 +223,326 @@ foam.CLASS({
               e.NEQ(foam.nanos.auth.Address.COUNTRY_ID, 'US'),
               e.REG_EXP(
                 foam.nanos.auth.Address.POSTAL_CODE,
-                /^^\d{5}(?:[-\s]\d{4})?$/i)
+                /^\d{5}(?:[-\s]\d{4})?$/i)
+            );
+          },
+          errorMessage: 'INVALID_POSTAL_CODE_ERR_MSG'
+        },
+        // Austria
+        {
+          args: ['postalCode', 'countryId'],
```

Comment:
validationPredicates - on each call we are walking them all?

Reply:
@jlhughes now you mention it, it doesn't look very efficient. I will wait for the response to the comment above and make changes if we can go with your suggestion above.